doc/articles/using-uno-ui.md

```diff
@@ -75,6 +75,27 @@ the **Show all Files** icon at the top.
 
 If you notice an issue, or an error in the commented code of the generated file, you may need to alter you Xaml.
 
+## Configure the Manifest for the WebAssembly Head
+In you WASM head, create a folder named `WasmScripts`, with a file named
+`AppManifest.js` with the `Embedded resource` build action.
+
+The manifest file can contain the following:
+
+```javascript
+var UnoAppManifest = {
+
+    splashScreenImage: "Assets/AppSplashScreen.scale-200.png",
+    splashScreenColor: "#3750D1",
+    displayName: "My Sample App"
```

Comment:
And if we want a localized version?

Reply:
No localization for now, and I'm thinking this file has high overlaps the PWA manifest file.